client/components/mma/upgrade/UpgradeSupportSwitchThankYou.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+	Button,
+	LinkButton,
+	Stack,
+	SvgCalendar,
+	SvgClock,
+	SvgEnvelope,
+} from '@guardian/source-react-components';
+import { useContext } from 'react';
+import { useNavigate } from 'react-router';
+import {
+	sectionSpacing,
+	signInContentContainerCss,
+	signInCss,
+	signInHeadingCss,
+	signInParaCss,
+} from '../../shared/SignIn';
+import {
+	headingCss,
+	stackedButtonLayoutCss,
+	whatHappensNextCss,
+} from '../cancel/cancellationSaves/SaveStyles';
```

Comment:
I think either we need to duplicate these styles or make them common somewhere else - feels odd to reference files from the switch and save journeys in this folder

Reply:
Yeah agreed. It's whether to start touching files which aren't part of this change or not

Reply:
In general I'd say it's fine, if it's just moving things around